### ISSUE TITLE
Click to load Disqus comments scriptlet not working 

### ISSUE THREAD

**Issue by troysjanda** [uBlockOrigin/uBlock-issues#773]
### URL(s) where the issue occurs
https://betanews.com

### Describe the issue
Click to load Disqus comments are not working, button is there but when clicked does not unblock comment section.

### Screenshot(s)

N/A (will not show in image the click or loading/ not loading process)

### Versions

- Browser/version: FF 70.0.1
- uBlock Origin version: 1.23.1b14
- Nano Defender

### Settings
All Ubo lists, Easylist/privacy 

All marked under setting/privacy

### Notes: conversation with gwarser on Ubo Subreddit

> Something broken in engine. @@||disqus.com/embed.js?_=1457540*$script is not matching in the middle of longer number. (In https://betanews.disqus.com/embed.js?_=1457540701460 for ex.)

> $ git bisect good && ./tools/make-firefox.sh 
> 7971b223855d3afae2ca57565f27da0631a1b045 is the first bad commit
> commit 7971b223855d3afae2ca57565f27da0631a1b045
> Author: Raymond Hill <rhill@raymondhill.net>
> Date:   Mon Oct 21 08:15:58 2019 -0400
> 
>     Expand bidi-trie usage in static network filtering engine
> 
>     Related issues:...

This was his workaround for the time being

> Workaround: add @@||betanews.disqus.com/embed.js?_=$script to My filters. [edit: or @@||disqus.com/embed.js?_=$script in general]
> 
> Can you create new issue on uBlock-issues? (copy my comments to it, I'm too sleepy)

**Comment by gwarser** [comment]
Did not noticed this was in uAssets, should be in uBlock-issues.

I almost forget to report this (was done internally), and after fix I forget to ping you back @troysjanda - sorry.

**Comment by gorhill** [comment]
> Fixed by gorhill/uBlock@7c0294b

However he said he is having the issue with 1.23.1b14.

**Comment by troysjanda** [comment]
> 
> 
> > Fixed by [gorhill/uBlock@7c0294b](https://github.com/gorhill/uBlock/commit/7c0294b)
> 
> However he said he is having the issue with 1.23.1b14.

Fix is working in 1.23.1b14

**Comment by troysjanda** [comment]
> 
> 
> Did not noticed this was in uAssets, should be in uBlock-issues.
> 
> I almost forget to report this (was done internally), and after fix I forget to ping you back @troysjanda - sorry.

No worries, I had almost forgotten to report it in the 1st place. I did start to file an issue in the uBlock-issues but since it was related to Filters I ended up making the report in Uassets. Glad it was fixed.

**Comment by Yuki2718** [comment]
Sorry for the duplicate. The scriptlet doesn't work on e.g.
`https://www.troyhunt.com/data-breach-disclosure-101-how-to-succeed-after-youve-failed/`
`https://abcnews.go.com/Health/coronavirus-map-tracking-spread-us-world/story?id=69415591&cid=clicksource_4380645_4_three_posts_card_hed`

**Comment by gorhill** [comment]
The original case here is still fixed, we should not re-open an issue which case is still fixed.

**Comment by Yuki2718** [comment]
> 
> 
> The original case here is still fixed, we should not re-open an issue which case is still fixed.

All right, thanks for re-opening my issue.
